projects/angular/src/accordion/stepper/stepper-panel.html

```diff
@@ -0,0 +1,64 @@
+<ng-container *ngIf="panel | async; let panel">
+  <div [ngClass]="getPanelStateClasses(panel)">
+    <div class="clr-accordion-header clr-stepper-header">
```

Comment:
```suggestion
    <div class="clr-stepper-header">
```

Can this just be the stepper class?

Reply:
No because `clr-accordion-header` class is responsible for coloring of `:hover`, `:active` and `opened` states. Which are same for bot components.

Alternativley we can add `clr-stepper-header` rule in `_accordion.clarity.scss`, but it doesnt look right to me.

Reply:
I think ideally at some point we completely separate the separate the stepper from the accordion. This PR is a good step in that direction. I would do anything not required for the design change in a separate ticket/PR though.